### DISCUSSION TITLE
安卓版，刚安装的，一直在获取配置，无法使用，也没法专业版登录账号，怎么办

### DISCUSSION THREAD

**Discussion by King4chaoren** [getlantern/lantern-forum-cn#820] [General]
![WechatIMG100](https://github.com/user-attachments/assets/d5e9cf28-e74f-4d70-a187-bba6745ceb82)
![WechatIMG99](https://github.com/user-attachments/assets/31584fb1-25c2-4876-a549-188f7cdb36fd)

**Comment by MaxMKMax** [comment]
@King4chaoren 您好. 请您提供我支付单据和账号邮箱. 我帮您提交问题上去.

**Reply by YcatZh** [reply]
什么时候解决问题，持续一星期了。

**Comment by MaxMKMax** [comment]
@King4chaoren 您好. 请您提供我支付单据和账号邮箱.

**Reply by glamoreye** [reply]
我遇到了同样的问题

**Reply by lsq88334753** [reply]
同样的问题，好多天了，新版本一直卡在获取配置，换了旧的版本8.2.0 VPN状态已连接但是server location显示N/A 无法使用，账号[453827835@qq.com](mailto:453827835@qq.com)

**Comment by Juichiro2049** [comment]
我也遇到了同样的问题，macos，8.2.6版本
